python/cuml/dask/ensemble/base.py

```diff
@@ -264,6 +287,17 @@ def _func_fit(model, input_data, convert_dtype):
     return model.fit(X, y, convert_dtype)
 
 
+def _func_predict(model, input_data, **kwargs):
+    X = concatenate(input_data)
+    with cuml.using_output_type("numpy"):
+        prediction = model.predict(X, **kwargs)

```

Comment:
Per discussion, for classification, this should probably be `predict_proba` and then we take the per-class average (maybe make a 3d array of nrows*nclasses*nworkers) and average over the 3rd dim. We'd also want to weight by the ntrees per worker, since they could differ a bit (e.g. training 101 trees on 10 workers would lead to an extra tree on 1). Same approach (weighted average reduction) should work for regression, just working like nclasses=1.